pages/data-visualization/install-and-connect.mdx

```diff
@@ -126,6 +126,39 @@ You should get the following result:
 
 </Steps>
 
+## Run Memgraph Lab with Docker
+
+Run the Memgraph Lab Docker image using the following command:
+
+```console
+docker run -d -p 3000:3000  --name lab memgraph/lab
```

Comment:
I'm following the command that we've introduced at https://memgraph.com/docs/getting-started/install-memgraph/docker#run-memgraph-lab-image. If needed, `-d` can also be removed from install docs.

Reply:
I think both works fine, it's just a question whether it's best practice to run in without detach and have logs in terminal or not, by default. I know that Lab has its logs in the UI, so that might be the reason for -d initially. It works like this for me as well, just wanted to make sure it is known that it works fine without detach. You can leave it as it is for now.